frontstage/common/authorisation.py

```diff
@@ -0,0 +1,26 @@
+from functools import wraps
+from flask import render_template
+from jose.jwt import encode, decode
+
+def jwt_authorisation(request):
```

Comment:
I prefer 'authorization' as well, just because that's what we're calling the cookie